src/phoneField/index.unit.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import PhoneField from 'phoneField'
+
+const countriesWhiteList = ['fr', 'es']
+const countriesWhiteListUpdated = ['de']
+
+const whiteListMapped = [
+  {"ariaLabel": "France", "label": "🇫🇷 +(33)", "value": "+33"},
+  {"ariaLabel": "Spain (España)", "label": "🇪🇸 +(34)", "value": "+34"}
+]
+const whiteListUpdateMapped = [
```

Comment:
It's not up to date right?

Reply:
No it's more like an other whiteList to test the state update when whiteList prop change.
I can rename it `whiteListMappedChange`

Reply:
Done